### PR TITLE
Add graphics

### PR DESCRIPTION
**Pull request by feeley**

### CODE REVIEW

File: include/actions.js [hunk 1]
```diff
@@ -883,15 +883,15 @@ CodeBootVM.prototype.execute2 = function (single_step) {
         }
         catch (e) {
             console.log(e);//TODO: remove
-            update_playground_visibility();
+            update_playground_visibility();//TODO: fix
```
Comment (belmarca) [inline include/actions.js:886]: Fix what?

File: include/codeboot.css [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 [data-cb-show-playground] .cb-playground {
     display: inline;
-    width: 384px;
+/*    width: 384px; */
```
Comment (belmarca) [inline include/codeboot.css:10]: Some superfluous (duplicates) comments.

File: include/codeboot.js [hunk 3]
```diff
@@ -1031,28 +1037,6 @@ CodeBootVM.prototype.setLargeFont = function (large) {
     }
 };
 
-CodeBootVM.prototype.setShowDrawingWindow = function (show) {
```
Comment (belmarca) [inline include/codeboot.js:1034]: The methods are used but removed from the prototype?

File: include/actions.js (hunk 1 above)
Comment (feeley) [reply include/actions.js:886]: a reminder for myself... it needs to be passed the `vm`

File: include/codeboot.js (hunk 3 above)
Comment (feeley) [reply include/codeboot.js:1034]: they no longer exist (replaced by `DrawingWindow.prototype.setShow`)

**Pull request merged**